scripts/run-figures.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
On https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/571, when I say

> Add a shell script that is designed to run _all steps_ required to get from the data download to the publication-ready figures either to the root of the repository or to `scripts`

Think about if the Strelka2 or Mutect2 file changed upstream and in the data download, for example. What steps or analysis modules would need to be run before you could make a mutational landscape figure that was entirely up to date? All of those steps need to be included.

Reply:
The rscript this calls should include all these steps

Reply:
I think you want those consensus steps here because the way you have it you've essentially hid away the part that's going to take a long time and require 128 GB of RAM. If I saw something that said `figure-scripts/fig2-mutational-landscape.R`, I would assume that the only thing in that was plotting.